### ISSUE TITLE
Memory Leak with JAMF Managed macOS Device

### ISSUE THREAD

**Issue by njb130** [igvteam/igv#989]
I'm running IGV 2.10.0 on macOS 11.4 that is managed through my institution using JAMF. Immediately upon opening the IGV app, the JAMFDaemon management tool begins to consume RAM until all physical memory is full and then begins to use up swap space (>20Gb total). This then leads to system crash.

This is specific to IGV, as other Java apps such as GSEA do not have any problems running. More importantly, it happens immediately upon loading IGV and does not require me to interact with the program at all to begin this behavior.

Has anybody else encountered this or something similar?

**Comment by njb130** [comment]
Closing this issue now, as this exact situation has been shown on the JAMF forum.
https://www.jamf.com/jamf-nation/discussions/39206/jamfdaemon-high-memory-usage

TL;DR: you must have Java installed separately

**Comment by pongorlorinc** [comment]
Thank you for the line njb130, this worked for me!
